Bindings/Java/tests/TestBasics.java

```diff
@@ -23,6 +23,12 @@ public static void main(String[] args) {
 
     m.addComponent(pj);
 
+    // Change from ArrayPtrs<Object*> being non-const const.
```

Comment:
Looks like this block was added to verify ArrayConstObjPtr, but doesn't really belong here. Did you mean to leave it in?